lib/loginUtils.js

```diff
@@ -11,28 +11,31 @@ export async function loadStoreData() {
 export async function lookupClerk(storeId, pin) {
   const currStore = await getStoresById(storeId);
   const clerks = await getClerksByIds(currStore.clerkIds);
-  let errorMsg = 'Invalid PIN';
+  let errorMsg = '';
+
   // No clerk at this store
   if (clerks.length === 0) {
-    return { status: status.NOT_FOUND, record: null, errorMsg };
+    errorMsg = 'No clerk found at this store';
```

Comment:
I'm not sure if we want this to be visible to the user (also I believe this is broken in master, probably because `clerkIds` doesn't exist on stores with no clerks) - @wangannie @12aschen ?

Reply:
Yeah I don't think it should be. Probably just a generic 'Invalid PIN' is safer.